src/ontology/mondo-edit.obo

```diff
@@ -412199,6 +412331,19 @@ synonym: "NPHS22" EXACT ABBREVIATION [OMIM:619155]
 xref: OMIM:619155 {source="MONDO:equivalentTo"}
 is_a: MONDO:0002350 {source="OMIM:619155"} ! familial nephrotic syndrome
 
+[Term]
+id: MONDO:0030896
+name: chromosome 13q33-q34 deletion syndrome
+xref: OMIM:619148 {source="MONDO:equivalentTo"}
+is_a: MONDO:0003847 ! Mendelian disease
```

Comment:
classify this more precisely

consider 

 MONDO:0016911  ! partial deletion of the long arm of chromosome 13

if not, at least chromosomal disease

need monochrom!

Reply:
reclassified as child of 'partial deletion of the long arm of chromosome 13'